css/free5e.scss

```diff
@@ -27,11 +27,19 @@ table {
   width: 100%;
 }
 
-.spell {
+.spell,.monster,.item {
   font-style: italic;
   color: #152347;
+}
 
-  &:after {
-    content: '\00a0🪄';
-  }
+.spell:after {
+  content: '\00a0🪄';
+}
+
+.monster:after {
+  content: '\00a0🧚';
+}
+
+.item:after {
+  content: '\00a0📦';
```

Comment:
This add a little package symbol to item names (that have been tagged accordingly) in the HTML output, similar to what I did for spells:

<img width="1006" height="178" alt="A small package symbol next to the text 'bag of holding'" src="https://github.com/user-attachments/assets/59b6af13-6994-4c20-84b2-ace8bca2c950" />